azure-devops/azext_devops/dev/repos/arguments.py

```diff
@@ -64,7 +64,21 @@ def load_code_arguments(self, _):
 
     with self.argument_context('repos policy merge-strategy') as context:
         context.argument('use_squash_merge', arg_type=get_three_state_flag(),
-                         help='Whether to squash merge always.')
+                         help='Whether to squash merge always. '
+                         'This option does not work for allowing other merge types.',
+                         deprecate_info=context.deprecate(redirect="--allow-squash"))
```

Comment:
How this is shown.. 

![image](https://user-images.githubusercontent.com/11888714/64332814-dc813000-cff2-11e9-8c5b-e3644ba6947e.png)